db/migrate/20230518161456_create_ingredients.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :ingredients do |t|
       t.string :name
       t.integer :cost
-
+      t.references :recipe, foreign_key: true
```

Comment:
The database had already been fully set up properly for you, no updates were needed.